src/command_modules/azure-cli-sql/azure/cli/command_modules/sql/_params.py

```diff
@@ -1144,17 +1143,25 @@ def _configure_security_policy_storage_params(arg_ctx):
                    help='The collation of the Azure SQL Managed Database collation to use, '
                    'e.g.: SQL_Latin1_General_CP1_CI_AS or Latin1_General_100_CS_AS_SC')
 
-    with self.argument_context('sql managed-db restore') as c:
+    with self.argument_context('sql midb restore') as c:
         create_args_for_complex_type(
             c, 'parameters', ManagedDatabase, [
                 'target_managed_database_name',
+                'target_managed_instance_name',
                 'restore_point_in_time'
             ])
+
         c.argument('target_managed_database_name',
                    options_list=['--target-db'],
                    required=True,
                    help='Name of the managed database that will be created as the restore destination.')
 
+        c.argument('target_managed_instance_name',
+                   options_list=['--target-mi'],
+                   help='Name of the managed instance to restore managed database to. '
+                   'This can be same managed instance, or another managed instance withing same resource group. '
```

Comment:
typo `withing` -> `within`

Is it not possible to specify another resource group? We have this for `sql db copy` command, see https://docs.microsoft.com/en-us/cli/azure/sql/db?view=azure-cli-latest#az-sql-db-copy

Reply:
Thanks! You are right, we can specify different rg. Fixed.